src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/getImageBlock.js

```diff
@@ -33,7 +33,7 @@ const getImageBlock = (type, blockData, isAmp) => {
   const { altText, dimensions } = imageData;
 
   return {
-    alt: htmlUnescape(altText),
+    alt: escape(altText),
```

Comment:
Do we even need to do this anymore, I remember @jamesdonoh making these changes that removed the need for escaping in block data (I think?) https://github.com/bbc/simorgh/commit/5d31ea66e902da5d5c348656d6e4c86b24a32b96 

I may be wrong but can we check we even need to escape?

Reply:
we did as of a week ago, this was a bug picked up in the alt text captions I think.

Reply:
Hmm, I must be missing something but seems a safe change